backend/src/books/books.service.ts

```diff
@@ -583,3 +583,29 @@ export const getLikeInfo = async (userId: number, bookInfoId: number) => {
   */
   return ({ "bookInfoId": 123, "isLiked" : false, "likeNum" : 15 });
 };
+
+export const updateBookInfo = async (book: types.UpdateBookInfo, bookInfoId: number) => {
+  let updateString = '';
+  const queryParam = [];
+  var bookInfoObject: any = {
+  } = book
+
+  for (let key in bookInfoObject) {
+    let value = bookInfoObject[key];
+    if (key !== '') {
+      updateString += `${key } = ?,`
+      queryParam.push(value)
+    } else if (key === null){
+      updateString += `${key} = NULL,`
+    } else {
+      // if (key === undefined) 
+    } 
+  }
+  updateString = updateString.slice(0,-1)
+  await executeQuery(`
+  UPDATE book_info 
+  SET 
+  ${updateString} 
+  WHERE id = ${bookInfoId}
+  `, queryParam);
+};
```

Comment:
```suggestion
  await executeQuery(`
      UPDATE book_info 
      SET 
      ${updateString} 
      WHERE id = ${bookInfoId}
      `, queryParam);
};
```

indent 를 한단계 더 들일것을 제안합니다.